libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/route/line/api/MapboxRouteLineApi.kt

```diff
@@ -249,6 +249,7 @@ class MapboxRouteLineApi(
         newRoutes: List<RouteLine>,
         consumer: MapboxNavigationConsumer<Expected<RouteLineError, RouteSetValue>>
     ) {
+        cancel()
```

Comment:
I'd like to mention that this API design risks starving the renderer entirely.

If we'd be in a situation where a route is continuously updated at an interval that's shorter than the route's processing time, we'd never draw a route because the cancelation calls would starve queue.

It's a very marginal risk for the system and use-case that we have in place but we might want to refactor this when opportunity arises.